android/src/ch/opengis/qfield/QFieldUtils.java

```diff
@@ -0,0 +1,122 @@
+package ch.opengis.qfield;
+
+import android.content.Context;
+import android.content.ContentResolver;
+import android.content.ContentUris;
+import android.net.Uri;
+import android.os.Build;
+import android.os.Environment;
+import android.provider.DocumentsContract;
+import android.provider.MediaStore;
+import android.database.Cursor;
+
+public class QFieldUtils {
```

Comment:
Wondering if we should call them something like FileSystemUtils or so?

Reply:
I don't have any strong feelings. I'd be inclined to leave it as is for now, but if you have a strong desire to see it rename, it can be arranged ;)

Reply:
It's an implementation detail, no strong desire